### DISCUSSION TITLE
unable to add compare_invoke.py file to setup.py

### DISCUSSION THREAD

**Discussion by vijaysagi1** [python-smpplib/python-smpplib#151] [Q&A] [answered]
Hi,

We downloaded the code trying to add custom .py files to the setup.py.

The following is the snapshot, 

    from setuptools import setup, find_packages
    import sys

    extra = {}
    if sys.version_info >= (3,):
    extra['use_2to3'] = True

    setup(name="project",
      version='1.0.1',
      description='SMPP library for python',
      packages=['project'],
      zip_safe=True,
      package_data={'': ['project/face.dat','project/testing.dat','project/compare_invoke.py']},
      classifiers=[
        'Development Status :: 4 - Beta',
        'Operating System :: OS Independent',
        'Programming Language :: Python',
        'Programming Language :: Python :: 2.6',
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.1',
        'Programming Language :: Python :: 3.2',
        'Topic :: Communications :: Telephony',
        'Intended Audience :: Telecommunications Industry',
        'License :: OSI Approved',
        ],
      **extra
    )

Once i ran the command **python setup.py install**  they are no errors but i am unable to import from python command prompt.  

I tried from python command line as **from compare_invoke import checkme**
error was 

> ImportError: No module named compare_invoke

Pls suggest

Thanks
vijay

**Comment by eigenein** [comment, marked as answer]
Hi Vijay,

I'm sorry, but this question is generic to `setuptools` and is not related to the package.

You'd rather seek help on a specialized resource like StackOverflow.

Regards,
Pavel
